ui/src/components/retreat/stream/StreamContainer.js

```diff
@@ -0,0 +1,106 @@
+import {
+  VStack,
+  Box,
+  Button,
+  Stack,
+  AspectRatio,
+  Link,
+  Text,
+  Container,
+  Flex
+} from '@chakra-ui/react';
+import { ChevronLeftIcon } from '@chakra-ui/icons';
+import OnlinePageButtons from './StreamButtons';
```

Comment:
lol why do u call it OnlinePageButtons if it's from StreamButtons? minor but made me a bit confused